src/Settings.h

```diff
@@ -33,12 +35,15 @@ class Settings
     // instead of the current palette.
     static QColor getDefaultColorValue(const std::string& group, const std::string& name, AppStyle style);
 
-    // User configuration file path
-    static QString userPreferencesFile;
+    // User settings file path
+    static QString userSettingsFile;
 
     // QSettings object
     static QSettings* settings;
 
+    // This works verify that the settings file provided by the user is a normal settings file
+    static bool verifyUserSettingsFile(const QString userSettingsFile);
```

Comment:
Consider calling this `isValidSettingsFile()`; it will give the developer the instant recognition that it returns a boolean.